README.md

```diff
@@ -52,42 +52,291 @@ With most carrying a footer containing:
+|--------------------------------|-------- |---------------------------------------------------------------------------------------------------------------------------------|
+| `players[]`                    | list    | The list of players.                                                                                                            |
+| `players[].id`                 | integer | The ID of the player.                                                                                                           |
+| `players[].login`              | string  | The username of the player.                                                                                                     |
+| `players[].number_of_games`    | integer | The number of games this player has played.                                                                                     |
+| `players[].country`            | string  | The two-letter [GeoIP ISO 3166](http://dev.maxmind.com/geoip/legacy/codes/iso3166/) country code of the player.                 |
+| `players[].clan`               | string  | The acronym of the clan of the player.                                                                                          |
+| `players[].global_rating`      | object  | The global rating of the player as an array with two values. The player rank to display can be calculated as `μ - 3σ`.          |
+| `players[].global_rating.mean` | float   | The mean (`μ`) value of the rating.                                                                                             |
+| `players[].global_rating.dev`  | float   | The deviation (`σ`) value of the rating.                                                                                        |
+| `players[].ladder_rating`      | object  | The 1v1 leaderboard rating of the player as an array with two values. The player rank to display can be calculated as `μ - 3σ`. |
+| `players[].global_rating.mean` | float   | The mean (`μ`) value of the rating.                                                                                             |
+| `players[].global_rating.dev`  | float   | The deviation (`σ`) value of the rating.                                                                                        |
+| `players[].avatar`             | object  | The avatar object of the player.                                                                                                |
+| `players[].avatar.url`         | string  | The avatar image URL (40px * 20px).                                                                                             |
+| `players[].avatar.tooltip`     | string  | The avatar tooltip text.                                                                                                        |
+
+
+
+
+
+
+## Social
+
+**TODO*** (Downlord:) Would it make sense to merge this into the login response?
+**TODO*** (Downlord:) `channels` and `autojoin` seem redundant.
```

Comment:
Channels is legacy; it used to be a list of channels the client should display for joining. Since we don't have these anymore it is indeed redundant. This information is available through the irc protocol as well.